spec/super_good/solidus_taxjar/api_params_spec.rb

```diff
@@ -221,27 +221,6 @@
       end
     end
 
-    context "when some line items have been returned" do
```

Comment:
WDYT, about leave this test as a regression spec to ensure that returned/cancelled items are not excluded?